Backend/prisma/schema.prisma

```diff
@@ -10,9 +10,13 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model users {
+model user  {
   id String @id @db.Uuid @default(uuid())
```

Comment:
I don't see you mention that `id` is the primary field. 
can you double check that?